### DISCUSSION TITLE
Custom "NxM Tiles"

### DISCUSSION THREAD

**Discussion by x1foideo** [rxhanson/RectanglePro-Community#41] [Q&A]
Hi,
Is there anyway for creating custom Tiles?
Explain Better:
I tried custom layouts for applications. Now imagine having 3 Chrome windows and want them
1) top left
2) top right
3) bottom

Well this doesn't seem to work. But Using tiles (2x2 or 2x3) yes.
So I'm asking is there someway to reproduce Tiles but in a custom Way?

**Comment by rxhanson** [comment]
At the moment, there is no way to do what you're looking for. I am planning on adding it in, but just need to think through the best way to make it in a way that's not confusing.

**Comment by x1foideo** [comment]
Alright.
If I didn't misunderstand and if I can, a good idea would be sth like where the user can drag the mouse in a NxM table (predefined or decided by him) and every time he releases the mouse it is considered as a window.
EG
<img width="623" alt="Schermata 2022-03-25 alle 08 32 05" src="https://user-images.githubusercontent.com/12828790/160075380-1fac9ced-6e4f-4052-a300-9366c2891f18.png">
I have this grid, I can drag the mouse over the first left col
<img width="623" alt="Schermata 2022-03-25 alle 08 32 05" src="https://user-images.githubusercontent.com/12828790/160075728-bebf76a9-fd1a-4911-97c5-4886980a9207.png">
And then the remeining dials:
<img width="623" alt="Schermata 2022-03-25 alle 08 32 05" src="https://user-images.githubusercontent.com/12828790/160075971-565c0981-6cd6-49c9-9c0e-4333f0271a7d.png">

Of course that's only an exemple with a 3x3 but you (or the user) could have a NxM grid to use.
I don't know if this is what you were talking about for the "not confusing".

**Reply by rxhanson** [reply]
This sounds like FancyZones on Windows. Is that what you're looking for?

**Reply by x1foideo** [reply]
Sincerely I never heard of it, but yes, practically it's it. 
However, with FancyZones i need to drag every window in a specific zone. (Don't know if there's this possibility in rectangle, but it would be really amazing)
Another idea could be creating a personal grid and then auto rearrange the windows on the desktop. As I described above a "custom NxM Tile" replacing the "2x2 Tile"

Anyway Both would be really useful

**Reply by rxhanson** [reply]
I do have plans to incorporate some sort of fancy zones behavior, but I'm not quite sure what the end result will be or when I'll have it out the door. I'll have to play around with things and see what I like.

**Reply by x1foideo** [reply]
Ok, hope to have this incredible feature in your incredible product. I really appreciate it.

Have a good day

**Comment by Heziode** [comment]
This feature makes me think of grid mode of [Moom](https://manytricks.com/moom/):

In fullscreen mode:
[Fullscreen mode](https://user-images.githubusercontent.com/16735076/193358348-14865c87-e499-499f-8762-f5c62944059e.mp4)

In hexagon mode:
[Hexagon mode](https://user-images.githubusercontent.com/16735076/193358357-f69cc16c-f605-42d9-be74-41ccca9263a2.mp4)

This is very handy, and we can choose the size of the grid. Could be a good addition
